komorebic/src/main.rs

```diff
@@ -852,6 +853,9 @@ enum SubCommand {
     /// Set the layout on the focused workspace
     #[clap(arg_required_else_help = true)]
     ChangeLayout(ChangeLayout),
+    // Cycle between available layouts
+    #[clap(arg_required_else_help = false)]
```

Comment:
```suggestion
    // Cycle between available layouts
    #[clap(arg_required_else_help = true)]
```

Probably this is required? @LGUG2Z